packages/help-center/src/components/help-center-more-resources.tsx

```diff
@@ -42,15 +44,17 @@ export const HelpCenterMoreResources = () => {
 				<li className="help-center-more-resources__resource-item help-center-link__item">
 					<div className="help-center-more-resources__resource-cell help-center-link__cell">
 						<a
-							href={ localizeUrl( 'https://wordpress.com/support' ) }
-							rel="noreferrer"
-							target="_blank"
-							className="help-center-more-resources__format-list-numbered"
-							onClick={ () => trackMoreResourcesButtonClick( 'support-documentation' ) }
+							href="/chat-history"
+							onClick={ ( e ) => {
+								e.preventDefault();
+								trackMoreResourcesButtonClick( 'support-history' );
+								navigate( '/chat-history' );
+							} }
```

Comment:
Should probably switch to `import { Link } from "react-router"` instead of an anchor with an onClick. If someone opens the link in a new tab it will be a 404.

Reply:
I've updated to use a Link component. We can still open the link in a new tab, though.